pkg/operator/v311_00_assets/bindata.go

```diff
@@ -782,6 +782,7 @@ metadata:
     route-controller-manager: "true"
 spec:
   # The number of replicas will be set in code to the number of master nodes.
+  replicas: 1
```

Comment:
@alvaroaleman thanks for the fix!

I think semantically it semantically it would be closer to set this to 0 since we need to set it later.

also might be worth it to do the nil check in the place where this was failing: https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/73dce4b8e92b992e75a45b23a08c1013211454f6/pkg/operator/sync_openshiftcontrollermanager_v311_00.go#L173

Reply:
The thinking here was that:
* By setting it in the manifest we can be sure that there won't be any NPDs now or later, even if there is or will be additional code that acceses this
* Setting it to zero might cause a change in behavior if there is any codepath that does not set the replicas

Reply:
> The thinking here was that:
> 
> * By setting it in the manifest we can be sure that there won't be any NPDs now or later, even if there is or will be additional code that acceses this
> * Setting it to zero might cause a change in behavior if there is any codepath that does not set the replicas

This is incorrect.  In the linked code, the value of `actualRCDeployment` is from client-go when an error happened, so you get a zero-value object. It doesn't have to do with this manifest's nil or not-nil value here.